### PR TITLE
November Newsletter

### PR DESCRIPTION
**Pull request by EdwinB12**

### CODE REVIEW

File: _newsletters/2022-11-25-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+### News, Web & Blogs
+
+#### New LunchBytes recording online
+
+* [LunchBytes Panel: Teaching Code](https://rse.shef.ac.uk/events/lunchbytes-2022-11-24.html) - As code and software become an ever bigger part of research (its likely that around 1/3 of researchers write code), coding and other digital skills become ever more important to researchers. Questions around teaching code were put to an online panel. The panel included [Emma Norling](https://www.sheffield.ac.uk/dcs/people/academic/emma-norling) (Director of Learning and Teaching in the Department of Computer Science at the University of Sheffield), [Shangshang Gu](https://www.linkedin.com/in/%E5%B0%9A%E5%B0%9A-%E9%A1%BE-89935b129) (School of Health and Related Research, University of Sheffield), [Ashley Cadby](https://www.sheffield.ac.uk/physics/people/academic/ashley-cadby) who plays a key role in teaching code in the Department of Physics and Astronomy at the University of Sheffield and [Norbert Gyenge](https://www.linkedin.com/in/norbert-g-gyenge-7ba718201) (IT Services) with expertise in shorter courses. Thanks to Bob for organising the event!
+
+#### News, Web and Publications
```
Comment (davidwyld) [inline _newsletters/2022-11-25-newsletter.md]: This heading is a bit repetitive of the one on line 17

File: _newsletters/2022-11-25-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+* [Intermediate Software Development Skills for Earth and Environmental Scientists:  Reproducible Research through Reusable, Reliable Code in Python](https://www.qualtrics.manchester.ac.uk/jfe/form/SV_87F94dIoT1aRwto) - Fancy improving your research techniques within the Earth, atmospheric and oceanic sciences? University of Manchester, 27–31 March 2023, in person. For any questions about the course and application, please contact Anja Le Blanc via e-mail at anja.leblanc@manchester.ac.uk.
+
+
+
+### Training
+
+#### Research IT Training
+
+Research IT is providing a place for beginners or advanced users to expand their knowledge of HPC and different programming languages. The courses are part of the Doctoral Development Programme. For more information please visit our training registration web page (via VPN): [crs.shef.ac.uk](https://crs.shef.ac.uk).
```
Comment (davidwyld) [inline _newsletters/2022-11-25-newsletter.md]: I don't know that this list of sold out courses deserves a place above some scheduled courses with places available, maybe we should move it down.

**Review by davidwyld — CHANGES_REQUESTED** [review]
Nice one, there is A LOT in this newsletter! Couple of suggestions.

File: _newsletters/2022-11-25-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+
+### Events
+
+* [Computing Insight UK 2022](https://www.scd.stfc.ac.uk/Pages/CIUK2022.aspx) - Manchester Central Convention Complex on Thursday 1 and Friday 2 December. The theme for the conference this year is "Sustainable HPC" with sub-themes including "Sustainable Skills", "Sustainable Funding" and "Net Zero Computing".
+* [PyData Global 2022](https://pdg22.wpengine.com) - 2022-12-01 to 2022-12-03, Online. The PyData Global conference is an opportunity to listen, learn, and share knowledge about best practices, new approaches, and emerging technologies for data management, processing, analytics, and visualization.
+* [University of Sheffield Inaugural Annual Open Research Lecture](https://www.eventbrite.co.uk/e/university-of-sheffield-inaugural-annual-open-research-lecture-tickets-440984976937) - Lecture given by Dr Tom Stafford titled 'Wide Open?'. Dr Tom Stafford will review concerns about Open Research practices: that they are unnecessary, performative, exclusive, neocolonial, a distracting example of data positivism, undermine scholarly societies, slow down research, generate confidentiality risks, amongst others.
```
Comment (Robadob) [inline _newsletters/2022-11-25-newsletter.md]: When?

File: _newsletters/2022-11-25-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+* [UK National Open Hackathon 2023](https://www.openhackathons.org/s/siteevent/a0C5e000005V6EDEA0/se000151) - Application Deadline January 16, 2023. Open Hackathons are multi-day intensive hands-on events designed to help computational scientists and researchers port and optimize their applications using GPUs.
+* [Free ARCHER2 course at Newcastle University covering Docker and Singularity](https://www.archer2.ac.uk/training/courses/221207-containers/) - This course is a run of the Docker and Singularity lessons in the Carpentries Incubator. 7-8th December.
+* [R course for Digital Humanities](https://n8cir.org.uk/events/r-digital-humanities/) - Following a similar format to the 2021 workshop, the course is a gentle introduction to R for text analysis. Over the course of two sessions you will be taught the basics of the powerful programming language before being provided with hands-on experience analysing long-form text in the RStudio development environment.
```
Comment (Robadob) [inline _newsletters/2022-11-25-newsletter.md]: When?

File: _newsletters/2022-11-25-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+* [Case study: Asynchronous task scheduling for the Music #AI project](https://casestudiesrcg.blogspot.com/2022/11/asynchronous-task-scheduling-for-music.html) - Research Innovation and IT have been working on an exciting project with researchers from the Department of Computer Science. The goal of this project was to improve the existing music AI web application in terms of usability, maintainability, and scalability.
+* [Code Reproducibility Training initiative](https://docs.google.com/document/d/1STVrPjk0djjavBIYIderU2rvD0lPXuHodey4O5KK6qU/edit#heading=h.y2o8qp1gtdb1) - The Code Reproducibility Training initiative which is part of the ELIXIR-CONVERGE WP2 project, aims to create training materials to learn how to develop sustainable and reproducible code.
+* [I-X Centre for AI in Science](https://ix.imperial.ac.uk/research-centre/aiscience/) -  Imperial’s I-X Centre for AI in Science. I-X is Imperial’s flagship AI initiative bringing AI together with interdisciplinarity and grand challenges. Our centre is dedicated to using AI to disrupt and advance Science, Engineering and Mathematics and is underpinned by core support from Schmidt Futures.
+
+
+### Events
```
Comment (Robadob) [inline _newsletters/2022-11-25-newsletter.md:35]: Would be nice to have a consistent date format

* Thursday 1 and Friday 2 December
* 2022-12-01 to 2022-12-03
* January 16, 2023
* 27–31 March 2023

etc

Comment (Robadob) [reply _newsletters/2022-11-25-newsletter.md:35]: Also they don;t appear to be in chronological order. Archer course is dec, but after something with a january deadline.

Comment (davidwyld) [reply _newsletters/2022-11-25-newsletter.md:35]: > Would be nice to have a consistent date format

This should just be a running comment on newsletters at this point..

File: _newsletters/2022-11-25-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+Research IT is providing a place for beginners or advanced users to expand their knowledge of HPC and different programming languages. The courses are part of the Doctoral Development Programme. For more information please visit our training registration web page (via VPN): [crs.shef.ac.uk](https://crs.shef.ac.uk).
+
+#### RSE Team: Git & GitHub through GitKraken workshop - from Zero to Hero!
+
+This is an introductory course, teaching the git and GitHub skills required to manage research code over it’s development lifecycle. These skills are essential for collaborative research teams. Please keep an eye out for new dates being released.
+
+This two half-day course is being run this month:
+
+* [28 November 2022 and 29 November 2022](https://rse.shef.ac.uk/training/workshop/2022-11-28-git-zero-hero)
```
Comment (Robadob) [inline _newsletters/2022-11-25-newsletter.md]: Not worth advertising this course, it's already started.

Comment (Robadob) [reply _newsletters/2022-11-25-newsletter.md]: Wasn't there a meeting last week to discuss new dates? (Not sure who to bother about this for more info, I just recall it being mentioned)

File: _newsletters/2022-11-25-newsletter.md (hunk 5 above)
Comment (ns-rse) [reply _newsletters/2022-11-25-newsletter.md:35]: ISO8601!

File: _newsletters/2022-11-25-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+* [Get up to 12 days of software engineering support for your HPC/ML/GPU project](https://rse.shef.ac.uk/collaboration/tier2/). The Research Software Engineering team is offering to support researchers with the use of High Performance Computing (HPC), Machine Learning (ML) and Graphics Processors (GPUs) in their research projects. All researchers at our University can receive up to £5,000 of support time, or roughly 12 days of support, provided by a Research Software Engineer (RSE). This call aims to increase the uptake of the University's Tier 2 HPC resources, which are national facilities available to researchers in all faculties. This is an open call and applications are considered on a first-come-first-served basis. There is no current deadline and applications will be reviewed on an ongoing basis.
+* [ARCHER2 Embedded CSE (eCSE) Support Call](https://www.archer2.ac.uk/ecse/calls/). The ARCHER2 eCSE call provides funding for up to 18 person months of an RSE/PDRA to work on codes to run on ARCHER2. The deadline for submitting documents for technical evaluations is 2022-10-04 16:00 BST with a final proposal deadline of 2022-10-25 16:00 BST.
```
Comment (Robadob) [inline _newsletters/2022-11-25-newsletter.md]: The deadline for this funding call has passed

File: _newsletters/2022-11-25-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+* [Wellcome PhD studentships](https://pheds-dtc.ac.uk/). Training the next generation of researchers in high-quality research of complex multi-component public health interventions and policies.
+* [PhD at University of Bristol: Open spectroscopy in Environmental Sciences: Machine-learningaugmented analysis and interpretation ](https://www.bristol.ac.uk/media-library/sites/earthsciences/documents/phd-projects/2023-start/Byrne_UOB_2023_Studentships.pdf) PhD that is associated with the Bristol RSE group. The project title is “Open spectroscopy in Environmental Sciences: Machine-learning- augmented analysis and interpretation”. The aim is to help develop machine learning models and software that can aid in the interpretation of spectra. Analysis of this data seeks to address specific environmental questions, such as how soil bacteria are connected to the release of greenhouse gases, or how changes in geochemistry influences the release of pollutants into drinking water.
```
Comment (Robadob) [inline _newsletters/2022-11-25-newsletter.md]: Deadline?

File: _newsletters/2022-11-25-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+* [Introduction to RNA-seq analysis in R](https://sbc.shef.ac.uk/training/rna-seq-in-r-2022-12-12/) - Monday 12th December 13:00 - 16:00, Wednesday 14th December 13:00 - 16:00, Friday 16th December 13:00 - 16:00
+
+
+### Opportunities
+
+#### Funding
+
+* [Artificial intelligence hubs for real data and for scientific and engineering research](https://www.ukri.org/opportunity/artificial-intelligence-hubs-for-real-data-and-for-scientific-and-engineering-research/) -  Apply for funding for a research hub in either artificial intelligence (AI) for real data or AI for scientific and engineering research.
+* [Mathematical and computational foundations of artificial intelligence](https://www.ukri.org/opportunity/mathematical-and-computational-foundations-of-artificial-intelligence/). Apply for funding to advance the fundamental mathematical, statistical, and computational concepts that underpin artificial intelligence (AI).
+* [Wellcome PhD studentships](https://pheds-dtc.ac.uk/). Training the next generation of researchers in high-quality research of complex multi-component public health interventions and policies.
```
Comment (Robadob) [inline _newsletters/2022-11-25-newsletter.md]: deadline?

File: _newsletters/2022-11-25-newsletter.md (hunk 1 above)
Comment (EdwinB12) [reply _newsletters/2022-11-25-newsletter.md]: How do we feel about 'Web and Publications' instead? Is that still too repetitive?

File: _newsletters/2022-11-25-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+
+* [Introduction to RNA-seq analysis in R](https://sbc.shef.ac.uk/training/rna-seq-in-r-2022-12-12/) - Monday 12th December 13:00 - 16:00, Wednesday 14th December 13:00 - 16:00, Friday 16th December 13:00 - 16:00
+
+
+### Opportunities
+
+#### Funding
+
+* [Artificial intelligence hubs for real data and for scientific and engineering research](https://www.ukri.org/opportunity/artificial-intelligence-hubs-for-real-data-and-for-scientific-and-engineering-research/) -  Apply for funding for a research hub in either artificial intelligence (AI) for real data or AI for scientific and engineering research.
+* [Mathematical and computational foundations of artificial intelligence](https://www.ukri.org/opportunity/mathematical-and-computational-foundations-of-artificial-intelligence/). Apply for funding to advance the fundamental mathematical, statistical, and computational concepts that underpin artificial intelligence (AI).
```
Comment (Robadob) [inline _newsletters/2022-11-25-newsletter.md]: Deadline?

File: _newsletters/2022-11-25-newsletter.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+If a course is "sold out" please join the wait list - we regularly email people to encourage those that can no longer attend to cancel.
+
+#### Bioinformatics Training
+
+* [Introduction to RNA-seq analysis in R](https://sbc.shef.ac.uk/training/rna-seq-in-r-2022-12-12/) - Monday 12th December 13:00 - 16:00, Wednesday 14th December 13:00 - 16:00, Friday 16th December 13:00 - 16:00
+
+
+### Opportunities
+
+#### Funding
```
Comment (Robadob) [inline _newsletters/2022-11-25-newsletter.md:80]: Similar to events, consistent date format and chronological ordering would be nice, makes it alot easier to scan.

File: _newsletters/2022-11-25-newsletter.md [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+* [CHIST-ERA Open & Re-usable Research Data & Software](https://www.chistera.eu/call-ord-announcement).  This call tackles the challenge of open research data and software from the perspective of their possible reuse. The objective is to create the conditions for research in any domain based on open or shared data and software. Call deadline: 2022-12-14 17:00 CET
+* [Oracle for Research Project Awards](https://www.oracle.com/research/project-awards/) Oracle Research (this years sponsors of the RSE conference) have an open grants program to sponsor research projects with free cloud credits to tackle computational challenges. The majority of cloud providers no longer offer free credits however Oracle grants include access to a variety of Oracle Cloud technologies including compute, GPUs, and HPC. In addition to cloud credits, they also provide researchers and supporting RSEs with technical cloud support to transition and optimize their workflows on an Oracle Cloud environment. If interested then apply directly.
```
Comment (Robadob) [inline _newsletters/2022-11-25-newsletter.md]: deadline?

File: _newsletters/2022-11-25-newsletter.md (hunk 9 above)
Comment (Robadob) [reply _newsletters/2022-11-25-newsletter.md]: Possibly a bit more context here too, expect phd studentships aren't of interest to most the readership (but what do I know someone submit it to the slack channel).

File: _newsletters/2022-11-25-newsletter.md (hunk 1 above)
Comment (davidwyld) [reply _newsletters/2022-11-25-newsletter.md]: Follow your heart.

File: _newsletters/2022-11-25-newsletter.md (hunk 6 above)
Comment (EdwinB12) [reply _newsletters/2022-11-25-newsletter.md]: We've decided some dates internally but not yet made the events. We decided it best not to advertise these yet until the eventbrite event has been created

File: _newsletters/2022-11-25-newsletter.md (hunk 5 above)
Comment (EdwinB12) [reply _newsletters/2022-11-25-newsletter.md:35]: Yup this is fair point, generally just copied from whatever format someone put in the Slack. Lazy Edwin! Anyway, I'll convert to ISO8601 and order chronologically

File: _newsletters/2022-11-25-newsletter.md (hunk 6 above)
Comment (Robadob) [reply _newsletters/2022-11-25-newsletter.md]: Could atleast hint that ones for months x/y/z will be announced soon, so keep an eye out on the mailing list.

File: _newsletters/2022-11-25-newsletter.md (hunk 12 above)
Comment (EdwinB12) [reply _newsletters/2022-11-25-newsletter.md]: Doesn't appear to be a deadline as far as I can tell. Worth putting something like 'Deadline N/A' or 'Open Call' or just leave blank?

File: _newsletters/2022-11-25-newsletter.md (hunk 7 above)
Comment (EdwinB12) [reply _newsletters/2022-11-25-newsletter.md]: Good spot, thanks

File: _newsletters/2022-11-25-newsletter.md (hunk 2 above)
Comment (EdwinB12) [reply _newsletters/2022-11-25-newsletter.md]: Yer thats fair, I'll move our two courses to the bottom of the training section

File: _newsletters/2022-11-25-newsletter.md (hunk 12 above)
Comment (Robadob) [reply _newsletters/2022-11-25-newsletter.md]: Yeah one of those will suffice

**Comment by EdwinB12** [comment]
Thanks for the reviews @Robadob @davidwilby , believe I've addressed the changes: 

1. Changed all dates to 2022-11-29 format 
2. Ensure all events have location and date as first bit of information
3. Moved events and funding to chronological order
4. Removed events and funding opportunities that have passed
5. Removed Git training date and added a note to look out for newly advertised course dates
6. Moved git and deep learning training to the bottom of the training section

**Review by Robadob — APPROVED** [review]
Looks good to me.

There's still a couple of dates in the Web and Pubs section, but probably fine in that context.

**Comment by EdwinB12** [comment]
Thanks @Robadob , I'll wait for Davids review before merging. Happy with a merge commit or do you want it squashing?

**Review by davidwyld — APPROVED** [review]

**Comment by Robadob** [comment]
> Thanks @Robadob , I'll wait for Davids review before merging. Happy with a merge commit or do you want it squashing?

Personally I'm a big fan of squashing then rebase + merge. As I like a linear history without junk split commits. But this isn't my repo, so not my problem.

**Pull request merged**